src/Compilers/CSharp/Portable/Binder/Semantics/Operators/BinaryOperatorOverloadResolution.cs

```diff
@@ -723,11 +723,17 @@ private void GetReferenceEquality(BinaryOperatorKind kind, ArrayBuilder<BinaryOp
             BoundExpression left,
             BoundExpression right,
             ArrayBuilder<BinaryOperatorAnalysisResult> results,
-            ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+            ref HashSet<DiagnosticInfo> useSiteDiagnostics,
+            bool skipNativeIntegerOperators = false)
```

Comment:
>skipNativeIntegerOperators [](start = 17, length = 26)

It feels a bit strange that this is computed in the caller, since it is part of the decision about whether an operator is a candidate, and all the needed data is already available here.